am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -136,86 +137,84 @@ public void revokeResourceAccess(@NotNull @Valid ExplicitAccessMetadata accessMe
      * @param userId    accessor ID
      * @param userRoles accessor roles
      * @param resource  envelope {@link Resource} and corresponding metadata
-     * @return envelope {@link FilterResourceResponse} with resource ID, filtered JSON and map of permissions if access
-     *         to resource is configured, otherwise null
-     * @throws PersistenceException if any persistence errors were encountered
+     * @return envelope {@link FilteredResourceEnvelope} with resource ID, filtered JSON and map of permissions if
+     *     access to resource is configured, otherwise null.
      */
-    @SuppressWarnings("PMD.AvoidLiteralsInIfCondition") // Magic number used until multiple roles are supported
-    @AuditLog("filtered access to resource '{{resource.resourceId}}' "
-        + "defined as '{{resource.type.serviceName}}|{{resource.type.resourceType}}|{{resource.type.resourceName}}' "
-        + "for accessor '{{userId}}' in roles '{{userRoles}}': {{result.permissions}}")
-    public FilterResourceResponse filterResource(@NotBlank String userId,
-                                                 @NotEmpty Set<@NotBlank String> userRoles,
-                                                 @NotNull @Valid Resource resource) {
-        if (userRoles.size() > 1) {
-            throw new IllegalArgumentException("Currently a single role only is supported. "
-                + "Future implementations will allow for multiple roles.");
-        }
-
+    @AuditLog("filtered access to resource '{{resource.id}} defined as '{{resource.definition.serviceName}}|"
+        + "{{resource.definition.resourceType}}|{{resource.definition.resourceType}} for accessor '{{userId}}' "
```

Comment:
A `resource.definition.resourceType` is bound twice.

Reply:
One more thing, there is no `'` before ` for accessor` which results in `defined as 'Service|Resource Type|Resource Type for` (no closing apostrophe)